src/vivarium/framework/values.py

```diff
@@ -311,13 +289,45 @@ def _call(self, *args: Any, skip_post_processor: bool = False, **kwargs: Any) ->
     def __repr__(self) -> str:
         return f"_Pipeline({self.name})"
 
+    @classmethod
+    def setup_pipeline(
+        cls,
```

Comment:
Does mypy not yell at this for `cls` not being type hinted? 

And related, do we need a docstring for the `cls` parameter below?

Reply:
`cls` is special like `self`. Neither one requires type-hinting or a docstring.